src/Microsoft.Health.Dicom.Blob/Features/Export/AzureBlobExportSinkProvider.cs

```diff
@@ -30,26 +27,42 @@ internal sealed class AzureBlobExportSinkProvider : ExportSinkProvider<AzureBlob
     public override ExportDestinationType Type => ExportDestinationType.AzureBlob;
 
     private readonly ISecretStore _secretStore;
+    private readonly IFileStore _fileStore;
+    private readonly IServerCredentialProvider _serverCredentialProvider;
     private readonly AzureBlobExportSinkProviderOptions _providerOptions;
+    private readonly AzureBlobClientOptions _clientOptions;
+    private readonly BlobOperationOptions _operationOptions;
     private readonly JsonSerializerOptions _serializerOptions;
     private readonly ILogger _logger;
 
     public AzureBlobExportSinkProvider(
-        IOptions<AzureBlobExportSinkProviderOptions> options,
-        IOptions<JsonSerializerOptions> serializerOptions,
+        IFileStore fileStore,
+        IServerCredentialProvider serverCredentialProvider,
+        IOptionsSnapshot<AzureBlobExportSinkProviderOptions> providerOptions,
+        IOptionsSnapshot<AzureBlobClientOptions> clientOptions,
+        IOptionsSnapshot<BlobOperationOptions> operationOptions,
+        IOptionsSnapshot<JsonSerializerOptions> serializerOptions,
         ILogger<AzureBlobExportSinkProvider> logger)
     {
-        _providerOptions = EnsureArg.IsNotNull(options?.Value, nameof(options));
+        _fileStore = EnsureArg.IsNotNull(fileStore, nameof(fileStore));
+        _serverCredentialProvider = EnsureArg.IsNotNull(serverCredentialProvider, nameof(serverCredentialProvider));
+        _providerOptions = EnsureArg.IsNotNull(providerOptions?.Value, nameof(providerOptions));
+        _clientOptions = EnsureArg.IsNotNull(clientOptions?.Get("Export"), nameof(clientOptions));
```

Comment:
nit: Should we make `Export` a const somewhere?

Reply:
Yeah, we should be consistent